webapp/components/wpt-results.html

```diff
@@ -258,6 +271,35 @@
         return /(\.(html|htm|py|svg|xhtml|xht|xml)$)/.test(path);
       }
 
+      computeSourcePath(path, manifest) {
+        if (!this.computePathIsATestFile(path) || !manifest) {
+          return path;
+        }
+        const itemSets = TEST_TYPES.map(t => manifest.items[t]).filter(i => i);
```

Comment:
Can you document that this is because the manifest can be missing some test types entirely? (Took me a while to work that out, and still not entirely sure.)

Reply:
Done.